bin/gnng

```diff
@@ -92,9 +92,15 @@ def fetch_router_list(args):
     blocked_groups = []
     if args:
         for arg in args:
-            if not pass_filters(nd.find(arg)):
+            # Try to find the device, but fail gracefully if it can't be found
+            try:
+                device = nd.find(arg)
+            except KeyError:
+                print 'Device not found: {}'.format(arg)
```

Comment:
I was thinking that the `device_match()` function could be used here instead.

```python
from trigger.netdevices import device_match
device = device_match('10')
# => "No matches for '10'."
```

See: http://trigger.readthedocs.org/en/latest/api/netdevices.html#trigger.netdevices.device_match

LMK